File: pages/StartPage.cs
namespace alttrashcat_tests_csharp.pages
{
    public class StartPage : BasePage
    {
        public StartPage(AltDriver driver) : base(driver)
        {
        }
        public void Load()
        {
            Log("StartPage: Loading scene");
            Driver.LoadScene("Start");
        }
        public AltObject StartButton { get => Driver.WaitForObject(By.NAME, "StartButton", timeout: 5); }
        public AltObject StartText { get => Driver.WaitForObject(By.NAME, "StartText", timeout: 5); }
        public AltObject LogoImage { get => Driver.WaitForObject(By.NAME, "LogoImage", timeout: 5); }
        public AltObject UnityUrlButton { get => Driver.WaitForObject(By.NAME, "UnityURLButton", timeout: 5); }
        public bool IsDisplayed()
        {
            Log("StartPage: Checking if displayed");
            if (StartButton != null && StartText != null && LogoImage != null && UnityUrlButton != null)
                return true;
            return false;
        }
        public void PressStart()
        {
            Log("StartPage: Pressing Start");
            StartButton.Tap();
        }
    }
}

File: pages/start_page.py
from alttester import By

from .base_page import BasePage


class StartPage(BasePage):
    def __init__(self, driver):
        super().__init__(driver)

    def load(self):
        self.log("StartPage: Loading scene")
        self.driver.load_scene("Start")

    @property
    def start_button(self):
        return self.driver.wait_for_object(By.NAME, "StartButton", timeout=5)

    @property
    def start_text(self):
        return self.driver.wait_for_object(By.NAME, "StartText", timeout=5)

    @property
    def logo_image(self):
        return self.driver.wait_for_object(By.NAME, "LogoImage", timeout=5)

    @property
    def unity_url_button(self):
        return self.driver.wait_for_object(By.NAME, "UnityURLButton", timeout=5)

    def is_displayed(self):
        self.log("StartPage: Checking if displayed")
        return all([
            self.start_button,
            self.start_text,
            self.logo_image,
            self.unity_url_button,
        ])

    def press_start(self):
        self.log("StartPage: Pressing Start")
        self.start_button.tap()
